tests/ui/run-functional-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+git clone --depth 1 https://github.com/mozilla/addons-server.git
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml up -d --build
+until docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml images | grep "addonsserver_addons-frontend_1" ;
+    do printf "."; sleep 1
```

Comment:
Oh, cool. This is nice and simple. It may need a timeout in case of unexpected errors but circle-ci will already have one so let's worry about that only if we need it.